src/utility/StorageUtility.js

```diff
@@ -1,172 +1,195 @@
-    {
-      regionName: ' Seattle',
-      regionId: 'SEA',
-      publicationDate: '2016-01-29',
-      expirationDate: '2016-12-05',
-      revisionNumber: 0,
-      chartUrl: 'https://www.google.com',
-      filePath: '',
-      isFavorited: false,
-    },
-    {
-      regionName: ' Seattle',
-      regionId: 'SEA',
-      publicationDate: '2016-01-29',
-      expirationDate: '2016-12-05',
-      revisionNumber: 0,
-      chartUrl: 'https://www.google.com',
-      filePath: '',
-      isFavorited: false,
-    },
-    {
-      regionName: ' Favorite 2',
-      regionId: 'SEA',
-      publicationDate: '2016-01-29',
-      expirationDate: '2016-12-05',
-      revisionNumber: 0,
-      chartUrl: 'https://www.google.com',
-      filePath: '',
-      isFavorited: true,
-    },
-    {
-      regionName: ' Seattle',
-      regionId: 'SEA',
-      publicationDate: '2016-01-29',
-      expirationDate: '2016-12-05',
-      revisionNumber: 0,
-      chartUrl: 'https://www.google.com',
-      filePath: '',
-      isFavorited: false,
-    },
-    {
-      regionName: ' Seattle',
-      regionId: 'SEA',
-      publicationDate: '2016-01-29',
-      expirationDate: '2016-12-05',
-      revisionNumber: 0,
-      chartUrl: 'https://www.google.com',
-      filePath: '',
-      isFavorited: false,
-    },
-    {
-      regionName: ' Seattle',
-      regionId: 'SEA',
-      publicationDate: '2016-01-29',
-      expirationDate: '2016-12-05',
-      revisionNumber: 0,
-      chartUrl: 'https://www.google.com',
-      filePath: '',
-      isFavorited: false,
-    },
-    {
-      regionName: ' Seattle',
-      regionId: 'SEA',
-      publicationDate: '2016-01-29',
-      expirationDate: '2016-12-05',
-      revisionNumber: 0,
-      chartUrl: 'https://www.google.com',
-      filePath: '',
-      isFavorited: false,
-    },
-    {
-      regionName: ' Favorite 3',
-      regionId: 'SEA',
-      publicationDate: '2016-01-29',
-      expirationDate: '2016-12-05',
-      revisionNumber: 0,
-      chartUrl: 'https://www.google.com',
-      filePath: '',
-      isFavorited: true,
-    },
-    {
-      regionName: ' Seattle',
-      regionId: 'SEA',
-      publicationDate: '2016-01-29',
-      expirationDate: '2016-12-05',
-      revisionNumber: 0,
-      chartUrl: 'https://www.google.com',
-      filePath: '',
-      isFavorited: false,
-    },
-    {
-      regionName: ' Seattle',
-      regionId: 'SEA',
-      publicationDate: '2016-01-29',
-      expirationDate: '2016-12-05',
-      revisionNumber: 0,
-      chartUrl: 'https://www.google.com',
-      filePath: '',
-      isFavorited: false,
-    },
-    {
-      regionName: ' Seattle',
-      regionId: 'SEA',
-      publicationDate: '2016-01-29',
-      expirationDate: '2016-12-05',
-      revisionNumber: 0,
-      chartUrl: 'https://www.google.com',
-      filePath: '',
-      isFavorited: false,
-    }
-  ];
-  
   return exampleVfrCharts;
 }
 
-export { getSavedCharts };
+const updateVfrCharts = (chartId) => {
```

Comment:
i was thinking more along the lines of permanent storage.  Cause the code in app container will not save after a reboot.  But i looked a async storage and that looks more like what i was thinking.